### PR TITLE
Added more content and addressed comments in pull #7

### PR DESCRIPTION
**Pull request by makoscafee**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -21,24 +21,39 @@ oppiaGithubPages.config(['$routeProvider', function($routeProvider) {
       controller: 'mainController',
       projectTag: 'A tool for creating interactive tutors'
     })
-    .when('/wiki/embedded', {
-      templateUrl: 'pages/embedded.html',
+    .when('/wiki/Embedded', {
```
Comment (seanlip) [inline app.js]: Do we ever use the non-wiki namespace (i.e. do we have any pointers to /_/_ where the first '*' is something other than 'wiki')? If not, maybe we don't need the '/wiki' part.

File: pages/wiki/embedded.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <!-- MAIN CONTENT -->
 <section id="main_content" class="inner">
   <h2>
-      <a id="embeded" class="anchor" href="#embeded" aria-hidden="true"><span class="octicon octicon-link"></span></a>Embed the exploration</h2>
+    <a id="embeded" class="anchor" href="#embeded" aria-hidden="true"><span class="octicon octicon-link"></span></a>Embed the exploration</h2>
```
Comment (seanlip) [inline pages/wiki/embedded.html]: Spelling: embedded

File: pages/wiki/theExplorationGallery.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!-- MAIN CONTENT -->
+<section id="main_content" class="inner">
+  <h2>
+  	<a id="The_Exploration_Gallery" class="anchor" href="#The_Exploration_Gallery" aria-hidden="true"><span class="octicon octicon-link"></span></a>The Exploration Gallery</h2>
```
Comment (seanlip) [inline pages/wiki/theExplorationGallery.html]: Indentation

File: app.js (hunk 1 above)
Comment (makoscafee) [inline app.js]: Done.I removed /wiki part

File: pages/wiki/embedded.html (hunk 2 above)
Comment (makoscafee) [inline pages/wiki/embedded.html]: Done.i corrected spelling error

File: pages/wiki/theExplorationGallery.html (hunk 3 above)
Comment (makoscafee) [inline pages/wiki/theExplorationGallery.html]: Done

**Pull request merged**